roofit/roofitcore/src/RooAbsMinimizerFcn.cxx

```diff
@@ -442,6 +449,36 @@ void RooAbsMinimizerFcn::printEvalErrors() const
    ooccoutW(_context, Minimization) << msg.str() << endl;
 }
 
+/// Apply corrections on the fvalue if errors were signaled.
+///
+/// Two kinds of errors are possible: 1. infinite or nan values (the latter
+/// can be a signaling nan, using RooNaNPacker) or 2. logEvalError-type errors.
+/// Both are caught here and fvalue is updated so that Minuit in turn is nudged
+/// to move the search outside of the problematic parameter space area.
+double RooAbsMinimizerFcn::applyEvalErrorHandling(double fvalue) const
+{
+   if (!std::isfinite(fvalue) || RooAbsReal::numEvalErrors() > 0 || fvalue > 1e30) {
```

Comment:
Can you explain in an inline comment where the magic number `1e30` comes from?

Reply:
As elaborated via mail, I cannot. It is a historical choice that I just copied over. Perhaps Wouter can elaborate.

Reply:
Note that this change happened in the single commit https://github.com/root-project/root/pull/15797/commits/ae8c120ceff8d114aff6ad028160d9b8b3978486 so it should be clear to future maintainers that this was indeed simply copied over. The origin of the number can be traced back through git history.

Reply:
Ah yes, that makes sense. I didn't realize at first that the code was only moved.